codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/SymbolMetadataProvider.kt

```diff
@@ -93,8 +93,10 @@ class BaseSymbolMetadataProvider(base: RustSymbolProvider) : SymbolMetadataProvi
     }
 
     companion object {
-        private val defaultDerives = with(RuntimeType) {
-            listOf(Debug, PartialEq, Clone)
+        private val defaultDerives by lazy {
```

Comment:
This also feels like a separate change?

Reply:
this should fix that error message you got where it just generically told you it couldn't be initialized and didn't actually say it was because Rust runtime was missing